src/datumaro/plugins/data_formats/coco/importer.py

```diff
@@ -18,9 +21,9 @@
     CocoPersonKeypointsBase,
     CocoStuffBase,
 )
-from datumaro.util.definitions import DEFAULT_SUBSET_NAME
+from datumaro.plugins.data_formats.coco.extractor_merger import COCOExtractorMerger
```

Comment:
```suggestion
from .extractor_merger import COCOExtractorMerger
```
It looks a little bit odd that the same module members are imported both via relative and fully-qualified paths. Probably, we can just use relative imports an all cases.